_includes/configuration/branding.md

```diff
@@ -1,8 +1,8 @@
 ## branding
 
-Use the **branding** option to disable the "**Powered by Tiny**" displayed in the status bar for [product attribution]({{site.baseurl}}/general-configuration-guide/attribution-requirements/).
+Use the `branding` option to disable the "**Powered by Tiny**" displayed in the status bar for [product attribution]({{site.baseurl}}/general-configuration-guide/attribution-requirements/).
 
-> Note: The "**Powered by Tiny**" product attribution is required for users on the Tiny Cloud Starter plan. Product attribution is **optional** for users on the Developer, Pro, or Custom packages.
+> Note: The "**Powered by Tiny**" product attribution is required for users on the Tiny Cloud Starter plan. Product attribution is optional for commercial users.
```

Comment:
The term "[commercial](https://ahdictionary.com/word/search.html?q=commercial)" implies that these users are using our product for profit. I am changing "commercial" to "premium", which is an industry accepted term for paying customers.
Regards,
Shikha

Reply:
makes sense